bindings/rust/wasmedge-sys/src/statistics.rs

```diff
@@ -56,3 +72,27 @@ impl Drop for Statistics {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_stat() {
+        // create a Statistics instance
+        let result = Statistics::create();
+        assert!(result.is_ok());
+        let stat = result.unwrap();
+
+        // check instruction count
+        assert_eq!(stat.instr_count(), 0);
+
+        // check instruction count per second
+        let count = stat.instr_per_sec();
+        assert!(count.is_nan());
+        // assert!(count, 0.0);
```

Comment:
Instruction per second  = Total executed instructions / total execution time.
If the statistics is not enabled or the total execution time is `0.0`. Then, the instruction per second should be `Nan` which represents something is divided by zero.

Reply:
For the statement, "If the statistics is not enabled ...", what does "enabled" mean? does it mean the statistics is enabled in a **VM** or something else? Thanks!

Reply:
The doc comment of the `instr_per_sec` function has been refactored by adding the description on the cases, in which `NaN` is returned. Thanks!

Reply:
You can enabled it in the Configure. And the VM/executor will save information to the Statistics module. However, if the VM/executor is not executed before, it should keep empty.

Reply:
@hydai You're right, `Config` has an option controlling it. Our motivation for this test code is just to verify a corner case, namely, instantiate an individual `Statistics`, and then verify the return values of the APIs of interest. Many thanks for your detailed explanation.